### ISSUE TITLE
Generate in-depth textual description of an element

### ISSUE THREAD

**Issue by MischaPanch** [penpot/penai#8]
Precursor to creating variations of elements. First steps towards this, other issues will follow

Will likely implement this by using a VLM that ingests a PNG. This issue is mostly about prompt engineering and evaluation of results.

Out of scope:
1. Considering the context of the containing penpot project
2. Use penpot data model or SVG as input (instead use PNG)
3. Integrate with retrieval from penpot backend (this issue is independent of the retrieval and can be done in parallel to it)
